src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/OkHttpWebSocketWrapper.java

```diff
@@ -30,7 +30,7 @@
     private WebSocketOnClosedCallback onClose;
     private CompletableSubject startSubject = CompletableSubject.create();
     private CompletableSubject closeSubject = CompletableSubject.create();
-    private final ReentrantLock closeLock = new ReentrantLock();
+    private final ReentrantLock stateLock = new ReentrantLock();
```

Comment:
A little unrelated to this pr, but do you know why we use ReentrantLock instead of [synchronized](https://docs.oracle.com/javase/tutorial/essential/concurrency/locksync.html)?

Reply:
I'm not sure there is a good reason. Maybe just oversight